.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]*"
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  release:
+    name: Release Starkup to Cloud Storage
+    runs-on: ubuntu-latest
+    steps:
+      - id: checkout
+        uses: actions/checkout@v4
+
+      - id: auth
+        uses: google-github-actions/auth@v2
+        with:
+          credentials_json: ${{ secrets.GCP_CREDENTIALS }}
+
+      - id: upload-file
+        uses: google-github-actions/upload-cloud-storage@v2
+        with:
+          project_id: ${{ secrets.PROJECT_ID }}
+          path: starkup.sh
+          destination: starkup.dev
+          process_gcloudignore: false
```

Comment:
I have an idea, but I'm not sure how feasible it is: we could try and install the script from the url in here, to verify we did not screw up acl or something. 

The thing I'm not sure how to handle best is to make sure we're using new script (doing curl straight away here could potentially lead to error or just an older version being installed). We probably could just wait a few seconds first, then install the script and check its version against the pushed tag.

thoughts?